library/src/test/java/SpecMathTest.java

```diff
@@ -164,4 +165,101 @@ void overlay_appliedToSpec_succeeds() throws IOException, UnexpectedTypeExceptio
 
     assertThat(actual).isEqualTo(expected);
   }
+
+  @Test
+  void filter_withSpecificPath_succeeds()
+      throws IOException, AllUnmatchedFilterException, UnionConflictException, UnexpectedTypeException {
```

Comment:
In these tests, its fine just to say throws Exception

Reply:
thanks, this will be done in a following PR (after these are merged) since there are many places where this can be changed.